operator/validator/event_handler.go

```diff
@@ -93,6 +93,7 @@ func (c *controller) handleOperatorRemovalEvent(
 	//	}
 	//}
 
+	// TODO: check by operator ID, not operator public key
```

Comment:
Just to note:
in order to achieve this, the controller should be aware of the operator ID.
In some cases (when the node is not synced events wise), the controller will not know the operator ID until the OperatorRegistrationEvent of the current operator will be handled.

Reply:
@olegshmuelov thanks